src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1274,8 +1274,11 @@ public void cannotSetStatusDate() throws Exception {
     JsonObject createdItem = getById(id).getJson();
     JsonObject initialStatus = createdItem.getJsonObject("status");
 
+    String createdDate = createdItem.getJsonObject("metadata").getString("createdDate");
+
+
     assertThat(initialStatus.getString("name"), is("Available"));
-    assertThat(initialStatus.getString("date"), nullValue());
+    assertThat(initialStatus.getString("date"), is(createdDate));
```

Comment:
Please can the name of this test be changed to reflect these changes

Reply:
Yep.